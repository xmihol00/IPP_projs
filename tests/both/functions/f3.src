.IPPcode21

DEFVAR GF@var
MOVE GF@var string@str1\010

CALL f3
WRITE TF@var
WRITE GF@var

DEFVAR GF@exit
MOVE GF@exit int@0
EXIT GF@exit

LABEL f3
CREATEFRAME
DEFVAR TF@var
MOVE TF@var string@str2\010
WRITE TF@var
PUSHFRAME
WRITE LF@var
POPFRAME
RETURN
