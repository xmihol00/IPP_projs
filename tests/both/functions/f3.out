str2
str2
str2
str1
